docs/plan/fix-ec-payment-event-on-browser-back.md

```
@@ -0,0 +1,567 @@
+- Vzor pro globální localStorage detekci existuje v `useAuthLoader.ts`
+
+## Desired End State
+
+Po implementaci:
+1. Event `ec.payment` se pushne při každém návratu z GoPay brány (úspěšném i neúspěšném)
+2. Retry count se správně inkrementuje i při použití tlačítka "zpět"
+3. Event obsahuje `isPaymentSuccessful: false` a `PaymentFalseReason: 'payment_abandoned'` při opuštění brány
+4. Řešení je robustní a funguje při návratu na jakoukoli stránku eshopu
+
+### Verifikace:
+- V GTM debuggeru se zobrazí `ec.payment` event při kliknutí na "zpět" z GoPay
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)